src/components/Restaurant.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Restaurant = () => {
+  return (
+    <div>
+      <h3>Restaurant Orders</h3>
+      <ul>
+        <Order orderType={"Pizzas"} />
+        <Order orderType={"Salads"} />
+        <Order orderType={"Chocolate cake"} />
```

Comment:
This isn't wrong exactly, but there is a shortcut if you are passing a string as a prop, you can instead write:

```js
<Order orderType="Chocolate cake" />
```

And React will understand that you are passing a string as a prop.